ro_help/static/css/bootstrap-social.css

```diff
@@ -0,0 +1 @@
#500a6f;border-color:rgba(0,0,0,.2)}.is-yahoo:hover{color:#fff;background-color:#500a6f;border-color:rgba(0,0,0,.2)}.is-yahoo.active,.is-yahoo:active,.open>.dropdown-toggle.is-yahoo{color:#fff;background-color:#500a6f;border-color:rgba(0,0,0,.2)}.is-yahoo.active.focus,.is-yahoo.active:focus,.is-yahoo.active:hover,.is-yahoo:active.focus,.is-yahoo:active:focus,.is-yahoo:active:hover,.open>.dropdown-toggle.is-yahoo.focus,.open>.dropdown-toggle.is-yahoo:focus,.open>.dropdown-toggle.is-yahoo:hover{color:#fff;background-color:#39074e;border-color:rgba(0,0,0,.2)}.is-yahoo.active,.is-yahoo:active,.open>.dropdown-toggle.is-yahoo{background-image:none}.is-yahoo.disabled.focus,.is-yahoo.disabled:focus,.is-yahoo.disabled:hover,.is-yahoo[disabled].focus,.is-yahoo[disabled]:focus,.is-yahoo[disabled]:hover,fieldset[disabled] .is-yahoo.focus,fieldset[disabled] .is-yahoo:focus,fieldset[disabled] .is-yahoo:hover{background-color:#720e9e;border-color:rgba(0,0,0,.2)}.is-yahoo .badge{color:#720e9e;background-color:#fff}.is-youtube{color:#fff;background-color:red;border-color:rgba(0,0,0,.2)}.is-youtube.focus,.is-youtube:focus{color:#fff;background-color:#de000a;border-color:rgba(0,0,0,.2)}.is-youtube:hover{color:#fff;background-color:#de000a;border-color:rgba(0,0,0,.2)}.is-youtube.active,.is-youtube:active,.open>.dropdown-toggle.is-youtube{color:#fff;background-color:#de000a;border-color:rgba(0,0,0,.2)}.is-youtube.active.focus,.is-youtube.active:focus,.is-youtube.active:hover,.is-youtube:active.focus,.is-youtube:active:focus,.is-youtube:active:hover,.open>.dropdown-toggle.is-youtube.focus,.open>.dropdown-toggle.is-youtube:focus,.open>.dropdown-toggle.is-youtube:hover{color:#fff;background-color:#801419;border-color:rgba(0,0,0,.2)}.is-youtube.active,.is-youtube:active,.open>.dropdown-toggle.is-youtube{background-image:none}.is-youtube.disabled.focus,.is-youtube.disabled:focus,.is-youtube.disabled:hover,.is-youtube[disabled].focus,.is-youtube[disabled]:focus,.is-youtube[disabled]:hover,fieldset[disabled] .is-youtube.focus,fieldset[disabled] .is-youtube:focus,fieldset[disabled] .is-youtube:hover{background-color:red;border-color:rgba(0,0,0,.2)}.is-youtube .badge{color:red;background-color:#fff}a.button{transition-timing-function:cubic-bezier(.25,.46,.45,.94);transition-property:background-color,transform;transition-duration:.3s;-webkit-transition-timing-function:cubic-bezier(.25,.46,.45,.94);-webkit-transition-property:background-color,transform;-webkit-transition-duration:.3s;-o-transition-timing-function:cubic-bezier(.25,.46,.45,.94);-o-transition-property:background-color,transform;-o-transition-duration:.3s;-moz-transition-timing-function:cubic-bezier(.25,.46,.45,.94);-moz-transition-property:background-color,transform;-moz-transition-duration:.3s}.is-instagram,.no-animation{-o-transition-property:none!important;-ms-transition-property:none!important;-moz-transition-property:none!important;-webkit-transition-property:none!important;transition-property:none!important}
```

Comment:
Is there any ways to do it without including a new css file?
Maybe just using bulma and some small styling?

Reply:
I don't have an answer for this, sadly 🤷‍♂ 

We could, probably, take from that `css` only the icons we want? I haven't seen `bluma` providing any type of social icons right out of the box but if there's a _cleaner_ way I'm ok looking into it.

Reply:
We will get the icons from the Design team. We can use dummy text for now. 
Let's not inlucde bootstrap

Reply:
I can update the PR as soon as icons are ready